Source/WebCore/platform/mac/SerializedPlatformDataCueMac.h

```diff
@@ -46,7 +46,7 @@ class SerializedPlatformDataCueMac final : public SerializedPlatformDataCue {
 
     WEBCORE_EXPORT SerializedPlatformDataCueValue encodableValue() const final;
 
-    WEBCORE_EXPORT static const HashSet<Class>& allowedClassesForNativeValues();
+    WEBCORE_EXPORT static const HashSet<RetainPtr<Class>>& allowedClassesForNativeValues();
```

Comment:
Do classes need to be retained and released? If so, this is great, but if not I want to make sure we are not adding any wasteful additional work.

This seems to take this pull request from "make a future compiler happy" to something even more valuable.

Reply:
This WebCore class change is identical to the one in WebKit, but it is forward-looking as WebCore isn't compiled with ARC yet.  Also, this removes another use of raw pointers in a `HashSet<>`.

I can remove the changes to `SerializedPlatformDataCueValue`, but it's (at minimum) inconsistent.

Reply:
Seems fine to change it.